### DISCUSSION TITLE
I need the tutor who is Ukrainian.

### DISCUSSION THREAD

**Discussion by Pioneer220** [community/community#139510] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hello everyone, I have developed many frontend projects and now needed for code review.
Please contact me if you are Ukrainian and have skills in web development.

**Comment by Oleksii9999** [comment, marked as answer]
hi, I live in Ukraine and can communicate& guide you with my skills. could you explain me about your project framework? i am familiar with Next.js/React.js

**Comment by Pioneer220** [comment]
Thanks for replying me. ;)
